python/tests/orderby_tests/orderby.py

```diff
@@ -0,0 +1,188 @@
+from tests.aggregate_tests.aggtst_base import TstView, TstTable
+
+
+class orderby_tbl(TstTable):
+    """Define the table used by the order by/limit tests"""
+
+    def __init__(self):
+        self.sql = """CREATE TABLE orderby_tbl(
```

Comment:
we need to test orderby with multiple columns too, perhaps it's easiest if you add multiple columns to this table? Use different types, and at least 3 columns, let's say VARCHAR and DECIMAL besides this one.

We will need to test order by for all our types, but adding columns with all types to a single table may be too much, so perhaps we'll also use one table for each type - but that can be a later PR.